truelearn/tests/test_utils_visualisations.py

```diff
@@ -0,0 +1,388 @@
+# pylint: disable=missing-function-docstring,missing-class-docstring
+import functools
+import random
+import pathlib
+import filecmp
+import types
+import os
+import sys
+from typing import Dict, Optional, Callable
+
+import pytest
+from matplotlib.testing.compare import compare_images
+
+from truelearn import learning, datasets, models
+from truelearn.utils import visualisations
+from truelearn.errors import TrueLearnTypeError
+
+
+BASELINE_DIR = (
+    pathlib.Path(__file__).parent / "baseline_files"
+)  # directory relative to truelearn/tests
+TMP_PATH = pathlib.Path("./tests")  # directory relative to .
+UUID = "71903f8e-0ae2-4fd1-9c0c-2290e95b21e9"  # a randomly chosen UUID, affect HTML gen
+# TODO: document bound
+BOUND = 3
```

Comment:
document this var, I put it outside of the function since its a constant

Reply:
I will rename it `bound`, document and restore it to its original position.